### ISSUE TITLE
Does the code have Settings for model saving

### ISSUE THREAD

**Issue by ansleepykid** [G-U-N/PyCIL#20]

**Comment by G-U-N** [comment]
Can you rephrase your problem more clearly? (Questions written in Chinese are also acceptable)

**Comment by G-U-N** [comment]
Do you mean how to save models trained after each incremental session? 

Please add `self.save_checkpoint(filename)` in the function `after_task` of the method you choose to run.

**Comment by Hephaestusxg** [comment]
> Do you mean how to save models trained after each incremental session?
> 
> Please add `self.save_checkpoint(filename)` in the function `after_task` of the method you choose to run.

![1](https://github.com/G-U-N/PyCIL/assets/137689049/5ed04ffe-5192-45a4-a835-0cf922f0fdbb)
![2](https://github.com/G-U-N/PyCIL/assets/137689049/e6c4f1dc-189f-4a77-83d3-ac42a30a944a)
![3](https://github.com/G-U-N/PyCIL/assets/137689049/c24e63b3-faa3-4ce9-96bb-ecf71528f2cd)
不好意思，我使用der的方法 在after_task后保存了权重，我发现这个方法模型的结构会动态的变化，全连接层也会改变，我根据变化的网络分别加载保存出来的权重 ，发现跟工具箱中的准确率都很大的区别，工具箱在初始训练7，增量1的条件下，基本能保持百分之90多，但是我自己单独拿出来测的话 在cifar10上测试，acc准确率很低。请问能帮我解决一下么
